go.mod

```diff
@@ -1,3 +1,7 @@
-module github.com/activeshadow/logr
+module github.com/JeremyMarshall/logr
```

Comment:
This file (`go.mod`) needs to be reverted back so the module name remains "github.com/activeshadow/logr". You can use your version w/ Logrus until this is merged by using the `replace` directive in `go.mod`.